packages/ruleset/src/functions/property-description.js

```diff
@@ -8,10 +9,15 @@ module.exports = function(schema, _opts, { path }) {
   return validateSubschemas(schema, path, propertyDescription);
 };
 
+const ruleId = 'property-description';
```

Comment:
I wonder if this gets passed in somewhere (like the `_opts`). Not that it would make a huge difference but it could alleviate the burden of maintaining the rule name in two different places

Reply:
From experimenting a little bit, I was able to find that the third argument passed to a rule function is essentially the entire "context" (this is where we retrieve the unresolved document when needed, etc.).  On this context object, there's a 'rule' field that contains info about the rule being invoked, and the 'name' field of that object contains the rule-id, as in this example:
```
  rule: Rule {
    name: 'schema-description',
    definition: {
      description: 'Schemas and schema properties should have a non-empty description',
      message: '{{error}}',
      given: [Array],
      severity: 'warn',
      formats: [Array],
      resolved: true,
      then: [Object]
    },
    owner: Ruleset {
      maybeDefinition: [Object],
      id: 1,
      formats: [Formats [Set]],
      definition: [Object],
      aliases: null,
      hasComplexAliases: false,
      extends: [Array],
      overrides: null,
      rules: [Object]
    },
    recommended: true,
    description: 'Schemas and schema properties should have a non-empty description',
    message: '{{error}}',
    documentationUrl: 'https://github.com/IBM/openapi-validator/blob/main/docs/ibm-cloud-rules.md#schema-description',
    resolved: true,
    formats: Formats(2) [Set] { [Function], [Function] }
  },

```
Notice that this Rule instance also has a field named 'documentationUrl' that has a very conveniently placed value that should point directly to its section in our reference docs :joy:.  I have no idea how that got there, but it's brilliant.  Weren't we just talking about something like this yesterday???

Reply:
Drats, I just double-checked that URL and it doesn't quite work because the tag value on the end should be prefixed with "rule-" (e.g. "rule-schema-description") :joy:

Reply:
Looks like we specify the URL in the ibm-oas.json file and spectral must be adding the `#<rule-id>` tag on the end.  I think that means we'll have to work out a way to avoid the "rule-" prefix on those tags inside our .md file.

Reply:
I'm not sure if we can/want to rely on the filename or not, but if we can maybe something like this could work too:
```js
const ruleId = path.parse(__filename).name;
```